content/en/docs/next/quickstart.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Quickstart
+weight: 900
+description: Get etcd up and running in less than 5 minutes!
+---
+
+Follow the instructions below to locally install, run, and test a simple single-member cluster of etcd.
+
+## Install etcd
+
+Set the environment variables to help with the commands that follow:
+
+```
+ETCD_VER={{< param git_version_tag >}}
```

Comment:
If it's possible, I would suggest to provide a stable release version here. As the release page says, v3.5.0-alpha.0 is an experimental release.

Reply:
Once we get this page into shape, as a part of this PR, I'd like to copy the page over to v3.4 -- which will render this line to v3.4.15. Also, when v3.5 is released, we'll update the `git_version_tag` to v3.5.X (whatever the release version will be).

Does this address your concern here?

Reply:
OK, that sounds good. Thanks!